### ISSUE TITLE
Xcode 7 and bitcode?

### ISSUE THREAD

**Issue by sean-hill** [leecrossley/cordova-plugin-apple-watch#25]
Hey I'm trying to compile for iOS 9, and I'm getting this error:

```
ld: '/<dir>/Plugins/cordova-plugin-apple-watch/libmmwormhole.a(MMWormhole.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64
```

Any suggestions?

**Comment by leecrossley** [comment]
I'm aware of this, there is an update coming (although I was holding off until iOS9 is officially released). Bitcode is required for watchOS apps unfortunately, so you can't disable.

**Comment by sean-hill** [comment]
Hey @leecrossley, that's great to hear. Now that the GM is released though, I'm trying to get my cordova app ready for iOS 9 and into review today. However, I can't get it to compile with this plugin involved. I either have to 1. Remove my apple watch app or 2. Have my users have to use a broken iOS app until this is resolved. Do you recommend any hacks to get it to work so I can get it into review today?

**Comment by leecrossley** [comment]
Give me an hour

**Comment by leecrossley** [comment]
I've done a release (v0.10.1) that only contains a fix for this (9fd0c581a3). Basically it will now compile MMWormhole with your app, as the src is included. I have given it a quick test and it works for me.

You will need to remove the old plugin (and libmmwormhole.a framework reference) and add the new version (also on npm).

Let me know if you have any issues.

**Comment by sean-hill** [comment]
You're the best, thanks @leecrossley.
